torch/_inductor/codegen/cpp_gemm_template.py

```diff
@@ -217,26 +222,30 @@ def add_choices(
         input_nodes,
         beta=1,
         alpha=1,
+        has_bias=False,
```

Comment:
Originally we use the number of input nodes to decide whether there is a bias (2: no bias, 3: with bias) but with the inputs from epilogue nodes as part of the template, there could be more inputs even if there is no bias. So we now use a dedicated flag to check that.